### ISSUE TITLE
Missing aria-role description for movable points (circle and graph)?

### ISSUE THREAD

**Issue by terracoda** [phetsims/trig-tour#129]
The Point on Circle and the Point on Graph are custom movable objects.

With VoiceOver, I hear "Point on Cirlce, group," or sometimes on first focus, "Point on Cirlce, group,, article." This makes me think there is no use of aria-roledescription in plave.

Custom objects can communicate their intended alt-input interaction with a good aria-roledecription.

For other movable object such as the balloons in BASE and the magnet in FL, we have simpley used, ```html aria-role-description="movable".

This should communicate intuitively to a learner that they can use any of the arrow keys to move the object. In the case of the circle, it may take a few key presses Up or Down, Left or Right to sort out clockwise and counter-clockwise movement around the circle. 

For the Point on Graph, learners will likely immediately go the the Left and Right Arrow keys, and this woul be reinforced if we made it clear in the description that it is a horizontal graph. 

@jessegreenberg, please add aria-roledescription="movable"

This should result in a focus event being read out as, "Point on Movable, movable" or sometimes, "Point on Circle, movable, article." It's annoying that it sometimes reads out the article parent. I do not know how to suppress that.

**Comment by terracoda** [comment]
Tagging @kathy-phet, @amanda-phet and @KatieWoe for information purposes.

**Comment by jessegreenberg** [comment]
Sounds good. We need a way to easily apply these options to all draggables so that we don't have to remember 'movable' in lots of sims.

**Comment by jessegreenberg** [comment]
OK, the attribute was added in the above commits. @terracoda would you like to verify?

**Comment by Nancy-Salpepi** [comment]
For https://github.com/phetsims/qa/issues/1198:
@jessegreenberg with VO when I tab to the point on the circle or the graph I don't hear their names. 

For the Point on the Circle:
<img width="321" alt="Image" src="https://github.com/user-attachments/assets/47d67dd8-2196-4e56-b73f-03fb6e804add" />

For the Point on the Graph:

<img width="480" alt="Image" src="https://github.com/user-attachments/assets/83f44b10-c686-4039-8241-e972a4baf87f" />

**Comment by jessegreenberg** [comment]
Thanks @Nancy-Salpepi, Ill investigate that.

**Comment by jessegreenberg** [comment]
I think this is fixed, @Nancy-Salpepi can you please verify that you hear the accessible name in VoiceOver?

**Comment by jessegreenberg** [comment]
Great, thanks! Ill check in with @terracoda on this.

**Comment by terracoda** [comment]
These are working great!

**Comment by Nancy-Salpepi** [comment]
https://github.com/phetsims/qa/issues/1211
For mobile VO, when I focus on the point on the circle or graph, it doesn't say 'moveable."

**Comment by jessegreenberg** [comment]
OK thanks @Nancy-Salpepi. It may be that iOS VoiceOver doesn't support aria-roledescription. Ill take a look.

**Comment by jessegreenberg** [comment]
@terracoda noted that aria-roledescription supported aria-roledescriptoin at one point.

**Comment by jessegreenberg** [comment]
I found this example: https://www.digitala11y.com/aria-roledescriptionproperties/ and tested it with iOS VoiceOver. iOS VoiceOver does not read the aria-roledescription.

**Comment by jessegreenberg** [comment]
According to this table: https://www.digitala11y.com/aria-roledescriptionproperties/#:~:text=The%20aria%2Droledescription%20property%20gives,or%20contains%20only%20whitespace%20characters.

iOS VoiceOver does not support aria-roledescription.

**Comment by jessegreenberg** [comment]
I will talk with @terracoda about how to proceed.

**Comment by jessegreenberg** [comment]
Some discussion with @terracoda on slack:

We tried another example to see if VO would work with a different HTML element:

```html
  <button
      tabindex="1"
      role="button"
      aria-label="Point on circle"
      aria-roledescription="movable"
  >Point on circle
  </button>
```

It did not.

I tried Chrome to see if that would work. It did not.

@terracoda found online that iOS VO is having trouble with roles since version 17.1 or 17.2. My testing was using 17.5.

I am going to add this to the list of accessibility bugs and create an issue in a11y research to submit a bug report to apple.
